assets/scss/partials/layout/article.scss

```diff
@@ -393,6 +393,14 @@
             top: 0;
             border: 0;
         }
+        & > video {
+            position: absolute;
+            width: 100%;
+            height: 100%;
+            left: 0;
+            top: 0;
+            border: 0;
+        }
     }
```

Comment:
Maybe we could just use `& > iframe, & > video` selector, since both share the same style.

Reply:
fix